### PR TITLE
Fix running on Windows

### PR DESCRIPTION
**Pull request by bjorn3**
`sys.getloadavg()` doesn't exist on Windows.

### CODE REVIEW

File: website/querylog.py [hunk 1]
```diff
@@ -22,11 +22,17 @@ def __init__(self, **kwargs):
             self.start_rusage = resource.getrusage(resource.RUSAGE_SELF)
         self.attributes = kwargs
         self.running_timers = set([])
-        self.set(
-            start_time=dtfmt(self.start_time),
-            pid=os.getpid(),
-            loadavg=os.getloadavg()[0],
```
Comment (rix0rrr) [inline website/querylog.py:28]: Your PR description says it's about `getrusage`, but it seems to be this is the line you're interested in, right? Is it that `os.getloadavg()` doesn't exist on Windows?

If that is the case, in order to reduce duplication somewhat, can you change it to the following instead:

```py
loadavg = os.getloavg()[0] if not IS_WINDOWS else None
self.set(
    # ...
    loadavg=loadavg
    )
```

Other than that... whoops, sorry. I forgot to check the docs apparently 😅

**Review by rix0rrr — CHANGES_REQUESTED** [review]

File: website/querylog.py (hunk 1 above)
Comment (bjorn3) [reply website/querylog.py:28]: Indeed. `getloadavg()` is the problem.

Comment (bjorn3) [reply website/querylog.py:28]: Applied your suggestion. Should I squash?

Comment (rix0rrr) [reply website/querylog.py:28]: We'll squash when we merge.

**Review by rix0rrr — APPROVED** [review]

**Comment by rix0rrr** [comment]
I think @Felienne needs to merge from `main` now...

**Comment by Felienne** [comment]
> I think @Felienne needs to merge from `main` now...

I can't push on @bjorn3's branch so he will have to merge main on his fork so I can merge

**Comment by bjorn3** [comment]
Rebased

**Comment by Felienne** [comment]
Thanks @bjorn3!!! Congrats on your first Hedy contribution!!

**Pull request merged**